.github/workflows/rust.yml

```diff
@@ -13,29 +13,17 @@ env:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
+    container:
+      image: public.ecr.aws/r5b3e0r5/3box/rust-builder:latest
```

Comment:
How was this image built?

Reply:
Just added a comment with the link https://github.com/3box/rust-builder

Reply:
Perfect, thanks.